File: world-beauty/src/WB-ATV1/src/Models/RG.ts
export default class RG {
    private valor: string;
    private dataEmissao: Date;

    constructor(valor: string, dataEmissao: Date) {
        this.valor = valor;
        this.dataEmissao = dataEmissao;
    }

    public get GetValor(): string {
        return this.valor;
    }

    public get GetDataEmissao(): Date {
        return this.dataEmissao;
    }
}

File: backend/WorldBeauty/RG.java
package WorldBeauty;

import java.util.Date;

public class RG {
    private String valor;
    private Date dataEmissao;

    RG(String valor, Date dataEmissao) {
        this.valor = valor;
        this.dataEmissao = dataEmissao;
    }

    public String GetValor() {
        return this.valor;
    }

    public Date GetDataEmissao() {
        return this.dataEmissao;
    }
}
